sympy/physics/continuum_mechanics/beam.py

```diff
@@ -432,27 +462,17 @@ def slope(self):
         >>> b.apply_load(R2, 30, -1)
         >>> b.apply_load(120, 30, -2)
         >>> b.bc_deflection = [(10, 0), (30, 0)]
-        >>> b.solve_for_reaction_loads(R1, R2)
+        >>> b.solve()
```

Comment:
Now as we are keeping the same name, you can change it back to `solve_for_reaction_loads`

Reply:
Yes, I think we shouldn't change the name.